frontend/src/components/pomodoroview/PomodoroView.jsx

```diff
@@ -123,41 +138,62 @@ function ClockSettings() {
     setWorkDurationMinutes,
     setBreakDurationMinutes,
   } = useContext(remainingTimeContext);
-
-  const handleChange = (variableSetter) => {
+  const handleDown = (value,changeValue) =>{
```

Comment:
use name (value, variableSetter) to keep in consistence with other variable naming convention. Although this convention might not be good and would be changed in the future refactoring